src/Sylius/Bundle/AdminBundle/Resources/config/routing/payment.yml

```diff
@@ -27,3 +27,29 @@ sylius_admin_payment_complete:
                 transition: complete
             redirect: sylius_admin_payment_index
             flash: sylius.payment.completed
+
+sylius_admin_payment_payment_request_index:
+    path: /payments/{id}/payment_requests
```

Comment:
Both of these routes can be extracted into a separate file and prefixed w/ `/payment/{id}/payment-requests/`